karma.conf.js

```diff
@@ -307,7 +307,14 @@ module.exports = (config) => {
       },
       plugins: [
         new DefinePlugin({
-          TEST_BASE_PATH: JSON.stringify(process.cwd() + argv.testBasePath)
+          TEST_BASE_PATH: JSON.stringify(process.cwd() + argv.testBasePath),
+          'process.browser': 'true',
```

Comment:
These lines are allowing the tests containing react-spectrum2 components to still run. This should be removed once we migrate the date picker to a react-spectrum3 component.